spec

```diff
@@ -4,12 +4,26 @@
 
 beginseg
     name "makerom"
-    address 0x8007F000
+    address 0x80000000
     include "build/asm/makerom/rom_header.o"
     include "build/asm/makerom/ipl3.o"
     include "build/asm/makerom/entry.o"
 endseg
 
+beginseg
+    name "framebuffer_1"
+    address 0x80000500
+    flags NOLOAD
+    include "build/src/buffers/framebuffer_1.o"
+endseg
+
+beginseg
+    name "pre_boot_buffer"
+    flags NOLOAD
+    // This segment is a just a dummy so we can symbolize this address range.
```

Comment:
```suggestion
    // This segment is just a dummy so we can symbolize this address range.
```

Reply:
Would "this segment is a placeholder so we can symbolize the address range" be any better/make sense?